src/Integration/LocalServices/RuleSetInspector.cs

```diff
@@ -36,7 +36,7 @@ internal class RuleSetInspector : IRuleSetInspector
         /// </summary>
         public const string DefaultVSRuleSetsFolder = @"Team Tools\Static Analysis Tools\Rule Sets";
 
-        public RuleSetInspector(IServiceProvider serviceProvider, params string[] userRuleSetDirectories)
+        public RuleSetInspector(IServiceProvider serviceProvider, params string[] knownRuleSetDirectories)
```

Comment:
Originally I though this will be the way I'll inject those directories but in practice it makes more sense to inject them as a method argument and ruse the instance instead of creating one instance per resolution (which would be the case otherwise).